lib/codecs/h264.js

```diff
@@ -210,10 +210,11 @@ H264Stream = function() {
         trackId: trackId,
         pts: currentPts,
         dts: currentDts,
-        data: data
+        data: data,
+        nalUnitType: data[0] & 0x1f
       };
 
-    switch (data[0] & 0x1f) {
+    switch (event.nalUnitType) {
```

Comment:
It's annoying to try and determine the type of nal unit if it's not one that we normally parse.

Reply:
so now, for like sei_rbsp, the nalUnitType would end up being `sei_rbsp` but others would just pass along the hex value of it?

Reply:
Yeah, thinking about it again I think I will change it to `nalUnitTypeCode` that way type comparisons won't break down the line, since we expect it to not exist or be a string.